examples/purge.py

```diff
@@ -26,13 +26,13 @@
     if len(versionlist) == 1:
         continue
     for index in range(len(versionlist) - 1):
-        print ("index is ".format(index))
+        print ("index is ", format(index))
         va = versionlist[index]
         components = hub.get_version_components(va)
         codelocations = hub.get_version_codelocations(va)
         # hub.execute_delete(va['_meta']['href'])
         print ("version {} has {} codelocations".format(va['versionName'], codelocations['totalCount']))
-        if codelocations > 0:
+        if codelocations['totalCount'] > 0:
```

Comment:
You could collapse the if and the for into the following I believe,

for codelocation in codelocations.get('items',[]):

Reply:
That change produces this error.. 

index is
version 7.6.32 has 2 codelocations
Traceback (most recent call last):
  File "/opt/il/blackduck-api/src/purge.py", line 35, in <module>
    for codelocation in codelocations.get['items', []]:
TypeError: 'builtin_function_or_method' object is not subscriptable

current solution works ok for now..